user-guide/Cloud_Platform/RemoteAccess/Accessing_DMS_remotely_with_Cube.md

```diff
@@ -15,6 +15,7 @@ From DataMiner 10.3.0/10.3.2 onwards, you can use DataMiner Cube to access your
 - You have been [granted access to dataminer.services features](xref:Giving_users_access_to_cloud_features).
 - You [have the DataMiner Cube desktop application installed](xref:Installing_configuring_the_DataMiner_Cube_software).
 - Your user account has been granted access to the DataMiner System.
+- You have the [APIGateway](xref:DataMinerExtensionModules#apigateway) installed and running.
```

Comment:
The way this is phrased now, it looks like APIGateway has to be installed on the client machine. Is this indeed what you mean, or do you mean that it must be installed on the DMA the user is trying to connect to?

Reply:
It is indeed required to be on the system not the client.